packages/waku/src/router/define-router.ts

```diff
@@ -629,3 +630,12 @@ export function unstable_defineRouter(fns: {
 
   return defineHandlers({ handleRequest, handleBuild });
 }
+
+let publicRouterConfig: any | undefined;
+
+export function unstable_getRouterConfig() {
+  if (!publicRouterConfig) {
+    throw new Error('Router config is not initialized yet');
+  }
+  return publicRouterConfig;
+}
```

Comment:
Feels like a hack. (We could technically define two routers.)

But, having this with `unstable_` prefix for experimenting things should be acceptable.

Reply:
Yes, I agree this is hack for sure, but the implementation wise, this is one direction I suppose?

A proper API to achieve this would require `unstable_defineRouter` or server entry convention itself to somehow expose such arbitrary metadata. The build metadata https://github.com/wakujs/waku/pull/1751 sounds fairly close but this metadata needs to be ideally accessible as early as possible for dev and ssg runtime.